notebooks/movie_subtitles_results-undersampling.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import numpy as np
+import re
+from string import punctuation 
+
+
+# In[2]:
+
+
+df=pd.read_csv("../data/raw/movies_meta.csv")
+
+
+# In[3]:
+
+
+df.head()
+
+
+# In[4]:
+
+
+df.shape
+
+
+# In[5]:
+
+
+df.title.to_csv('../models/Titles.csv',index=False, header=False)
+
+
+# In[6]:
+
+
+df_sub=pd.read_csv("../data/raw/movies_subtitles.csv")
```

Comment:
Can you include these files in the relevant folders? `Titles.csv` should also be under data/raw and maybe rename it to `movies_titles.csv` for consistency.

Reply:
Yes. I have them in the raw folder which I put in .gitignore

Reply:
The Titles.csv is the one (which I will remove later) I made to see what are the titles of the movies available.